### ISSUE TITLE
Titles are not displayed properly

### ISSUE THREAD

**Issue by aashish108** [sissbruecker/linkding#233]
I see titles like this, can anything be done here? It looks like characters not being decoded from their HTML codes or something?

```
[Elon Musk Says &#39;Own Physical Things&#39; but Won&#39;t Sell Bitcoin, Doge](https://markets.businessinsider.com/news/currencies/elon-musk-own-physical-things-wont-sell-bitcoin-ether-doge-2022-3)
```

**Comment by sissbruecker** [comment]
Is this an imported bookmark, or did you add that bookmark manually, using the scraped title? I just tested adding this bookmark manually and had no problems. For importing the bookmark file though I'm not sure if that uses a specific character encoding.

**Comment by aashish108** [comment]
Yeah its an imported bookmark. Its only a few bookmarks here and there though so not a major but its still noticible.
